a3p-integration/proposals/z:acceptance/package.json

```diff
@@ -1,16 +1,30 @@
 {
   "agoricProposal": {
-    "type": "/agoric.swingset.CoreEvalProposal"
+    "type": "/agoric.swingset.CoreEvalProposal",
+    "sdk-generate": [
+      "testing/start-valueVow.js start-valueVow",
+      "testing/restart-valueVow.js restart-valueVow"
+    ]
   },
   "type": "module",
   "license": "Apache-2.0",
   "dependencies": {
+    "@agoric/internal": "0.3.3-dev-5676146.0",
     "@agoric/synthetic-chain": "^0.1.0",
+    "@cosmjs/stargate": "^0.32.3",
+    "@cosmjs/tendermint-rpc": "^0.32.3",
+    "@endo/errors": "^1.2.2",
+    "@endo/far": "^1.0.4",
+    "@endo/init": "^1.0.4",
+    "agoric": "0.21.2-dev-5676146.0",
```

Comment:
I didn't realize we had dependencies from a3p onto agoric-sdk (besides the bundles being generated). I'm wondering if there isn't a better way to handle this than previously published sdk dev versions. Would a relative path work here? Or a workspace link?